android-patches/patches/Dialog/ReactAndroid/src/main/java/com/facebook/react/modules/dialog/PlatformAlertFragment.java

```diff
@@ -1,97 +1,97 @@
 --- "d:\\code\\work\\rn-62-db\\ReactAndroid\\src\\main\\java\\com\\facebook\\react\\modules\\dialog\\PlatformAlertFragment.java"	1969-12-31 16:00:00.000000000 -0800
 +++ "D:\\code\\work\\rn-62-d\\ReactAndroid\\src\\main\\java\\com\\facebook\\react\\modules\\dialog\\PlatformAlertFragment.java"	2020-05-02 19:01:45.601710900 -0700
 @@ -0,0 +1,93 @@
-+/**

```

Comment:
Not sure why this file has changes .. instead of just moving ..

Actually, this "dialog" patches are no longer applied .. We apply only these : BasicBuild RNHostBase V8Integration Focus

Others can be deleted for now ...